runtime/tchannel_client.go

```diff
@@ -219,8 +219,14 @@ func (c *tchannelOutboundCall) finish(err error) {
 }
 
 func (c *tchannelOutboundCall) logFields() []zapcore.Field {
+	var hostPort string
+	if c.call != nil {
```

Comment:
The problem here is not `c.call` is `nil`, it's the `c.call.RemotePeer()` being nil, in that case we can set host to `unknown` instead of empty string, because empty string might cause the key to not show in the logs at all.

Reply:
we discussed this offline, c.call is indeed nil.